apps/fabric-website/src/pages/HomePage/HomePage.base.tsx

```diff
@@ -0,0 +1,266 @@
+            <p>{this._renderLink('#/get-started', 'Get started', false, true)}</p>
+          </div>
+        </div>
+      </section>
+    );
+  };
+
+  private _renderPlatformCardsSection = (): JSX.Element => {
+    const { isMountedOffset } = this.state;
+    const { theme, styles } = this.props;
+
+    const platformKeys = Object.keys(platforms);
+    const lastPlatform = platformKeys.length - 1;
+    const beforeColor = platforms[platformKeys[0]].color;
+    const afterColor = platforms[platformKeys[lastPlatform]].color;
+
+    const classNames = getClassNames(styles, {
+      theme,
+      isMountedOffset,
+      beforeColor,
+      afterColor,
+      isInverted: true
+    });
+
+    return (
+      <div className={classNames.platformCardsSection}>
+        <div className={classNames.inner}>
+          <div className={classNames.card} style={{ background: platforms.web.color }}>
+            <Icon iconName="WebLogo-homePage" className={classNames.cardIcon} />
+            <h3 className={classNames.cardTitle}>Web</h3>
+            <ul className={classNames.cardList}>
+              <li className={classNames.cardListItem}>{this._renderLink('#/styles/web', 'Styles')}</li>
+              <li className={classNames.cardListItem}>{this._renderLink('#/controls/web', 'Controls')}</li>
+              <li className={classNames.cardListItem}>{this._renderLink('#/get-started/web', 'Get started')}</li>
+            </ul>
+          </div>
+          <div className={classNames.card} style={{ background: platforms.ios.color }}>
+            <Icon iconName="AppleLogo-homePage" className={classNames.cardIcon} />
+            <h3 className={classNames.cardTitle}>iOS</h3>
+            <ul className={classNames.cardList}>
+              <li className={classNames.cardListItem}>{this._renderLink('#/controls/ios', 'Controls')}</li>
+            </ul>
+          </div>
+          <div className={classNames.card} style={{ background: platforms.android.color }}>
+            <Icon iconName="AndroidLogo-homePage" className={classNames.cardIcon} />
+            <h3 className={classNames.cardTitle}>Android</h3>
+            <ul className={classNames.cardList}>
+              <li className={classNames.cardListItem}>{this._renderLink('#/controls/android', 'Controls')}</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    );
+  };
+
+  private _renderPlatformsSection = (): JSX.Element => {
+    const { isMountedOffset } = this.state;
+    const { theme, styles } = this.props;
+
+    // We need to get classNames within this method for offset transitions and inverted sections.
+    const classNames = getClassNames(styles, {
+      theme,
+      isMountedOffset,
+      isInverted: true
+    });
+
+    return (
+      <section className={classNames.platformsSection}>
+        <div className={classNames.sectionContent}>
+          <div className={classNames.oneHalf}>
+            <h2 className={classNames.platformsTitle}>Build across platforms</h2>
```

Comment:
Can consider "cross-platform" instead of "across platform", but this copy text existed already in the diff